### PR TITLE
[Hacktober Submission]

### PR DESCRIPTION
**Pull request by CrimsonTome**
Created website development section including a list of static site generators

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,10 +17,11 @@
 9. [Learning Resources](#education)
 10. [Linux Distributions](#linux)
 11. [Electronics](Electronics.md)
-12. [Hypervisors and virtual machines](Virtualisation.md#virtualisation)
+12. [Web Development](#web)
```
Comment (closebracket) [inline README.md]: Hi @CrimsonTome, 

Thanks for making a submission to Hacktoberfest. I think this category might be better suited under Programming on Line 12. Could you add it as a subcategory and move the resources to the programming file, please?

File: README.md [hunk 2]
```diff
@@ -153,6 +154,19 @@ If you're not sure how to contribute, please follow **[this handy tutorial](http
 6. [Tails](https://tails.boum.org/index.en.html) - A portable distribution that runs only in memory, for the ultimate privacy protection.
 7. [Qubes OS](https://www.qubes-os.org/intro/) - A reasonably secure operating system. _Recommended by Edward Snowden_
 8. [Raspberry Pi OS](https://www.raspberrypi.org) - For use with the range of Raspberry Pi single board computers. A great distribution if you're looking to get into programming (and don't mind spending a bit of money on the hardware to run it), especially involving Python or electronics (the Pis can come with GPIO pins for use with electronics). There are some beginner projects listed on the site but there are a lot of projects made by the Pi community.
+## Web Development <a name="web"></a>
```
Comment (closebracket) [inline README.md]: Interesting set of resources, some of which I'd never heard of! Good job!

File: README.md (hunk 1 above)
Comment (closebracket) [reply README.md]: Don't forget to update the contents of each file. Really we should look at a template and import it into each.

**Review by closebracket — CHANGES_REQUESTED** [review]

**Comment by CrimsonTome** [comment]
Hi @closebracket 

Sure, I can move them to programming, I'll get it done when I'm next at a pc, by the end of today for sure.

**Comment by closebracket** [comment]
> 
> 
> Hi @closebracket
> 
> Sure, I can move them to programming, I'll get it done when I'm next at a pc, by the end of today for sure.

Thank you.

**Comment by CrimsonTome** [comment]
moved web development static site generators from their own section to programming.md under static site generators 
@closebracket I hope that is sufficient.

**Review by closebracket — APPROVED** [review]
Whilst the navigation changes weren't made, they are not essential for the Hacktoberfest submission. I will do these in my next update. You've added some good resources here @CrimsonTome, well done! Keep contributing!

**Pull request merged**